x-pack/plugins/security_solution/server/lib/detection_engine/scripts/post_rule.sh

```diff
@@ -24,7 +24,7 @@ do {
   -u ${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD} \
   -X POST ${KIBANA_URL}${SPACE_URL}/api/detection_engine/rules \
    -d @${RULE} \
-  | jq .;
+  | jq -S .;
```

Comment:
I am fine with this. I think for consistency we should update everywhere but I won't inflict that on this PR. You can leave it here and we will do a sweep through to do a sort on all the fields when doing POST, GET, DELETE, later...

Reply:
Whoops -- left that accidentally... 😬  I've made a note to update the rest to use it as well in `Part II`. Was really helpful in testing/debugging via the CLI scripts -- would like to turn on the linter rule for alphabetizing params too as that caused a bit of fatigue through all of this as well!